src/store/modules/facility/actions.ts

```diff
@@ -744,10 +744,15 @@ const actions: ActionTree<FacilityState, RootState> = {
     stateGroups = stateGroups.filter((group: any) => !facilityGroupIds.includes(group.facilityGroupId))
 
     try {
-      const facilityCountByGroup = await FacilityService.fetchFacilityCountByGroup(facilityGroupIds)
+      const facilityCountByGroup = await FacilityService.fetchFacilityCountByGroup(JSON.parse(JSON.stringify(facilityGroupIds)))
```

Comment:
Why we have deep-cloned this value?

Reply:
facilityGroupIds is list of ids which gets empty at the end of below api call. We again want's to use this facilityGroupIds in another api hence deep cloning it.